src/components/MoneyRequestReportView/MoneyRequestReportActionsList.tsx

```diff
@@ -539,7 +539,7 @@ function MoneyRequestReportActionsList({
             style={[styles.flex1]}
             ref={wrapperViewRef}
         >
-            {shouldUseNarrowLayout && !!selectionMode?.isEnabled && (
+            {shouldUseNarrowLayout && !!selectionMode && (
```

Comment:
do we still need `!!`?
Would be cool if `selectionMode` was always a `boolean` (unless it can't be - but then maybe you can handle it inside hook to make it always `boolean`)